libraries/provider_rightscale_volume.rb

```diff
@@ -837,16 +849,7 @@ def get_next_device(exclusions = [])
 
             last_device_letter_in_use = $1
 
-            if node['cloud']['provider'] == 'ec2' && device_type == 'hd'
-              # This is probably HVM
-              hvm = true
-              # Root device is /dev/hda on HVM images, but volumes are xvd in /proc/partitions,
-              # but can be referenced as sd also (mount shows sd) - PS
```

Comment:
It is safe because the ec2 provider is caught in an ealier [condition](https://github.com/rightscale-cookbooks/rightscale_volume/pull/19/files#diff-0f450d3b6be7ae8ef8870385dcb222fcR817), and this removed block changes the device_type from `hd` to `xvd`, which is now set to `xvd` [explicitly](https://github.com/rightscale-cookbooks/rightscale_volume/pull/19/files#diff-0f450d3b6be7ae8ef8870385dcb222fcR825)